src/main/java/com/slimebot/games/games/Wordchain.java

```diff
@@ -0,0 +1,212 @@
+            }
+        }
+
+        character = content.toCharArray()[content.length()-1];
+        nextTurn(lastWord);
+    }
+
+    @Override
+    public void onButtonInteraction(ButtonInteractionEvent event) {
+        String id = event.getButton().getId();
+        if(!id.startsWith(uuid.toString()))return;
+        id = id.replace(uuid +":", "");
+
+        switch(id) {
+            case "join":
+                if(!join(event.getMember().getIdLong())) {
+                    event.reply(":x: Du bist schon in einem Game!").setEphemeral(true).queue(); // TODO: better message
+                    return;
+                }
+                event.editMessageEmbeds(
+                        new EmbedBuilder()
+                                .setColor(Main.embedColor(String.valueOf(guildId)))
+                                .setTitle(event.getMessage().getEmbeds().get(0).getTitle())
+                                .setDescription("Um zu erfahren wie \"Wortkette\" funktioniert nutze ```/wordchain explanation```") // TODO
+                                .addField("Spielleiter:in:", "<@"+gameMaster+">", true)
+                                .addField(":timer: Timeout:", seconds + "s", true)
+                                .addField(":x: Max Fehler:", lives + " Fehler", true)
+                                .addField("Spieler:", player.stream().map(p -> ("<@"+ p + ">")).collect(Collectors.joining("\r\n")), true)
+                                .setFooter("GameID: " + uuid)
+                                .setTimestamp(Instant.now())
+                                .build()
+                ).queue();
+                event.reply("<@"+ event.getMember().getId()+"> ist dem Spiel beigetreten!").queue(); // TODO: better message
+                return;
+        }
+
+        if(event.getMember().getIdLong() != gameMaster) {
+            event.reply(":x: Du bist nicht Spielleiter:in! uiashdfuihwesuiafgh").queue(); // TODO: better message
+            return;
+        }
+        switch(id) {
+            case "settings:create":
+                create();
+                return;
+            case "start":
+                if(player.size() >= 2) {
+                    event.getMessage().delete().queue();
+                    start();
+                }else {
+                    event.reply(":x: Es müssen mindestens 2 Spieler:innen mit machen!").setEphemeral(true).queue();
+                }
+                return;
+
+        }
+    }
+
+    @Override
+    public void onStringSelectInteraction(StringSelectInteractionEvent event) {
+        String id = event.getSelectMenu().getId();
+        if(!id.startsWith(uuid.toString()))return;
+        id = id.replace(uuid +":", "");
+
+        if(event.getMember().getIdLong() != gameMaster) {
+            event.reply(":x: Du bist nicht Spielleiter:in! uiashdfuihwesuiafgh").queue(); // TODO: better message
+            return;
+        }
+        switch(id) {
```

Comment:
Handle these in the command itself. The easiest way would be to use the `ui` module.